tests/integration/bond_test.py

```diff
@@ -387,3 +397,11 @@ def test_create_vlan_over_a_bond(bond99_with_slave):
     ) as desired_state:
         assertlib.assert_state(desired_state)
     assertlib.assert_state(bond99_with_slave)
+
+
+def test_create_vlan_as_slave_of_bond(eth2_vlan102, bond99_with_slave):
+    bond_name = bond99_with_slave[Interface.KEY][0][Interface.NAME]
+    vlan_id = 102
+    slaves = ['{}.{}'.format(bond_name, vlan_id)]
```

Comment:
Please, use `"` instead of `'`.

Reply:
If I am not wrong, the slave name should match the VLAN name. Let me know if I am missing something.